### ISSUE TITLE
Ignore any third-party theme that uses "default" as ID

### ISSUE THREAD

**Issue by dacap** [aseprite/aseprite#4226]
From ticket: https://igarastudio.zendesk.com/agent/tickets/5545

We must limit/ignore any theme extension that wants to replace the default theme and is not the official one installed in the data directory (this might be harder to detect in portable versions of Aseprite).

**Comment by Gasparoken** [comment]
I was able to reproduce the crash. The `aseprite-theme` corresponds to an old version of Aseprite, some "styles" were missing. The crash was due to some style pointers being nullptr.
So @dacap what is your opinion on the following approach?:

If any style pointer is nullptr, the aseprite startup process is suspended and an alert is displayed:
`The "aseprite-theme" is corrupted or outdated; Delete this file and reinstall Aseprite.`

Alternatives:
- Calculate checksum of `aseprite-theme` files and compare with a precalculated checksum. If it does not match -> same alert as above.
- Regenerate `aseprite-theme` files every time Aseprite is started (from code embedded within the binary).
